include/arch/arm/aarch32/cortex_a_r/timer.h

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2021 IoT.bzh
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_ARCH_ARM_AARCH32_CORTEX_A_R_TIMER_H_
+#define ZEPHYR_INCLUDE_ARCH_ARM_AARCH32_CORTEX_A_R_TIMER_H_
+
+#ifdef CONFIG_ARM_ARCH_TIMER
+
+#ifndef _ASMLANGUAGE
+
+#include <drivers/timer/arm_arch_timer.h>
+#include <sys/device_mmio.h>
+#include <zephyr/types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define ARM_ARCH_TIMER_BASE     DT_REG_ADDR_BY_IDX(ARM_TIMER_NODE, 0)
+#define ARM_ARCH_TIMER_IRQ      ARM_TIMER_VIRTUAL_IRQ
+#define ARM_ARCH_TIMER_PRIO     ARM_TIMER_VIRTUAL_PRIO
+#define ARM_ARCH_TIMER_FLAGS    ARM_TIMER_VIRTUAL_FLAGS
+
+#define TIMER_CNT_LOWER 0x00
+#define TIMER_CNT_UPPER 0x04
+#define TIMER_CTRL      0x08
+#define TIMER_ISR       0x0c
+#define TIMER_CMP_LOWER 0x10
+#define TIMER_CMP_UPPER 0x14
+
+#define TIMER_IRQ_ENABLE    BIT(2)
+#define TIMER_COMP_ENABLE   BIT(1)
+#define TIMER_ENABLE        BIT(0)
+
```

Comment:
nit: can you indent all these defines at the same level?

Reply:
Ok will do